<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Nidhi" name="Guru99 logindemo1" tests="1" failures="1" timestamp="3 Sep 2020 14:55:44 GMT" time="16.905" errors="0">
  <testcase name="loginpage_test" time="14.263" classname="tests.TC01_Login_test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at tests.TC01_Login_test.loginpage_test(TC01_Login_test.java:39)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- loginpage_test -->
</testsuite> <!-- Guru99 logindemo1 -->
